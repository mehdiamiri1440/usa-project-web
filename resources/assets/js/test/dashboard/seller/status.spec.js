import { mount } from '@vue/test-utils'
import Status from '../../../components/dashboard/seller/dashboard/status.vue'
import axios from 'axios';
// // Mock out all top level functions, such as get, put, delete and post:
// jest.mock("axios");

// jest.mock('axios', () => {
//     return {
//         post: jest.fn(() => Promise.resolve({ data: {} })),
//     };
// });


// jest.mock('../../../components/dashboard/seller/dashboard/status.vue', () => ({
//     template: '<div>Mock it out</div>'
// }));

// test('should mount Status without crashing', () => {
//     const wrapper = mount(Status, {
//         stubs: ['router-link']
//     })
//     // console.log(wrapper);
//     expect(wrapper).toMatchSnapshot()
// })

