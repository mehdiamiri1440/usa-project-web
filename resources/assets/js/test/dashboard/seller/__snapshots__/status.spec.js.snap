// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should mount Status without crashing 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
